examples/prometheus/prometheus.yaml

```diff
@@ -619,6 +619,26 @@ objects:
           action: keep
           regex: openshift-template-service-broker;apiserver;https
 
+      # Scrape config for the service catalog
+      - job_name: 'openshift-service-catalog'
+        scheme: http
+
+        kubernetes_sd_configs:
+        - role: pod
+
+        relabel_configs:
+        - source_labels: [__meta_kubernetes_namespace, __meta_kubernetes_pod_name]
```

Comment:
Since you're only looking for pods in the kube-service-catalog namespace, it would be better to tell it upfront in the k8s sd configuration rather than use relabeling:

```
      - job_name: 'openshift-service-catalog'
        scheme: http

        kubernetes_sd_configs:
        - role: pod

        namespaces:
          names:
          - kube-service-catalog

        relabel_configs:
        - source_labels: [__meta_kubernetes_pod_name]
          action: keep
          regex: controller-manager-(.+)
```

I've got https://github.com/openshift/origin/pull/17683 pending for fixing the existing jobs but it needs a rebase.

Reply:
Great, that makes sense, thanks for the details.  I've updated the config accordingly.